### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by zvacet**

### CODE REVIEW

**Comment by jleclanche** [comment]
This is fine to land.

**Comment by pmattern** [comment]
In `lxqt-config-file-associations_hr.ts` tag `language` isn't set in line 3, some translations are missing and some remain tagged `unfinished` for apparently no reason.
Also, one trailing dot is missing in line 67. This is basically a non-issue which nevertheless makes QtLinguist suspect something horrible has happened.

What again puzzles me is that all these problems could have been avoided by following the advices we've been discussing rather extensively in lxde/compton-conf#22.
It's pretty obvious that you're spending a lot of time trying to help improve LXQt which is much appreciated so it would be good for both you and us if we could get these problems fixed. Git can be a bit of beast sometimes. If you need further advice feel free to ask on the GitHub issue tracker, the mailing list on http://sourceforge.net/projects/lxde or IRC channel freenode #lxde.

For now you can probably solve the problems stated above as follows:
* Edit the file and fix the problems, possibly using QtLinguist.
* Merge the changes into the previous commit by running `git commit --amend` leaving the commit message as is by just quitting the editor that comes up.
* Push the changes to your GitHub repo by running `git push --force zvacet patch-2:patch-2` where `zvacet` is the name given to your repo locally as can be seen in the first column of the output of `git remote --verbose`, e. g.    
`$ git remote --verbose`   
` projekt https://github.com/lxde/lxqt-config.git (fetch)`   
` projekt https://github.com/lxde/lxqt-config.git (push)`   
`zvacet  https://github.com/zvacet/lxqt-config.git (fetch)`   
`zvacet  https://github.com/zvacet/lxqt-config.git (push)`   
(So may vary on your system and e. g. be "origin").   
This will get this PR updated as well.

File: liblxqt-config-cursor/translations/lxqt-config-cursor_hr.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <name>WarningLabel</name>
+    <message>
+        <location filename="../ui/warninglabel.ui" line="14"/>
+        <source>Form</source>
+        <translation>Forma</translation>
+    </message>
+    <message>
+        <location filename="../ui/warninglabel.ui" line="44"/>
+        <source>LXQt could not find any cursor theme. The default X11 cursor theme will be used instead. LXQt looked in the following directories:</source>
+        <translation>LXQt ne može naći nijednu temu pokazivača. Biti će korištena zadana X11 tema pokazivača.LXQt je gledao u slijedećedirektorije:     </translation>
```
Comment (philippwiesemann) [inline liblxqt-config-cursor/translations/lxqt-config-cursor_hr.ts]: A space is missing between second and third sentence and there are unnecessary trailing spaces.

File: lxqt-config-file-associations/translations/lxqt-config-file-associations_hr.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.0" language="hr_HR">
```
Comment (philippwiesemann) [inline lxqt-config-file-associations/translations/lxqt-config-file-associations_hr.ts]: This could be "hr" (instead of "hr_HR") to match the file name.

**Pull request closed without merging**